ssp

```diff
@@ -3947,17 +3701,23 @@ sub check_for_huge_logs {
         '/usr/local/apache/logs/suphp_log'        => {},
         '/var/cpanel/secdatadir/ip.pag'           => { help => 'Apache using a lot of CPU for no good reason? Try moving aside and restarting Apache. See EA-4092.' },
         '/var/named/data/named.run'               => {},
-        '/var/cpanel/eximstats_db.sqlite3'        => { size => 5_000_000_000 },
+        '/var/cpanel/backups/metadata.sqlite'     => { info_size => 1_000_000_000, warn_size => 10_000_000_000 },
```

Comment:
I would suggest leaving out any /var/cpanel/backups/metadata.sqlite related operations at this point as we have some super heavy logic coming in.

The map is essentially going to be like this on a per-version topology:
1) 11.70.0.0 -> 11.70.0.15 ( metadata V2 - in /var/cpanel/backups/metadata.sqlite )
2) 11.70.0.15 -> 11.70.0.18 ( metadata V3 - in /var/cpanel/backups/metadata.sqlite )
3) Once a new Candidate hits it will be Metadata V3 but the pathing will be $BACKUPDIR/.metadata.sqlite ( where $BACKUPDIR is set in /var/cpanel/backups/config

We are also going to be adding in METADATA_DISABLE into /var/cpanel/backups/config as well so we will need to spec out a routine to handle this.

TECH-505

I suppose for now the way this is written is accurate and we can stage further changes to this functionality via TECH-505

Reply:
All this does is point out the size of the file if it is large.  Until those changes actually go live, this will be immediately useful.